code/modules/shuttle/emergency.dm

```diff
@@ -193,9 +193,26 @@
 	return shuttle.hijack_status
 
 /obj/machinery/computer/emergency_shuttle/AltClick(user)
+	if(isalien(user))
+		xeno_queen_action(user)
+		return
 	if(isliving(user))
 		attempt_hijack_stage(user)
```

Comment:
Could you redefine (and possibly rename) `attempt_hijack_stage()` instead of adding an istype check here?